ISSUE_TEMPLATE/NEW-TASK.yaml

```diff
@@ -0,0 +1,50 @@
+name: New Task
+description: New task
+title: "[Task]: "
+labels: []
+type: task
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Describe the task
+      placeholder: What needs to be done?
+      value: "A task needs to be done!"
+    validations:
+      required: true
+  - type: dropdown
```

Comment:
Is it possible to make a "list of checkboxes" instead of a dropdown, to remove a button click?

Reply:
Yearh - will merge this for now to test. But we can definitely change to checkboxes if prefered